docs/06_how-to-guides/40_multi-index/how-to-define-a-singleton.md

```diff
@@ -87,6 +121,7 @@ class [[eosio::contract]] singleton_example : public contract {
 And below is a possible implementation for the two `get` and `set` actions defined above. It also demonstrates the usage of a couple of singleton methods. Note that the `set` action makes use of the singleton's `set` method, for which parameter is the account to pay for the new value stored. In this case, the same account name that is stored in the primary value is the payer. However, it can be a different account if so required.
```

Comment:
And below is a possible implementation for the to - start with And - can't do this

Here are example implementations of ...

Reply:
It also demonstrates the usage of a couple of singleton methods. => which methods

Reply:
Note that the `set` action makes use of the singleton's `set` method, for which parameter is the account to pay for the new value stored. => or which parameter is the account to pay for the new value stored => this is not clear

Reply:
However, it can be a different account if so required. Don't think this is necessary